StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RightAccessoryButton.swift

```diff
@@ -43,12 +43,7 @@ extension RowButton {
         private var label: UILabel {
             let label = UILabel()
             label.text = accessoryType.text
-            switch accessoryType {
-            case .edit, .viewMoreChevron, .update:
-                label.font = appearance.scaledFont(for: appearance.font.base.medium, style: .footnote, maximumPointSize: 20)
-            case .viewMore:
-                label.font = appearance.scaledFont(for: appearance.font.base.medium, size: 14, maximumPointSize: 20)
```

Comment:
They should be the same size (they are in the figma), I'm assuming this was an attempt to work around the issue I fixed here